beginner_source/flava_finetuning_tutorial.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+"""
+TorchMultimodal Tutorial: Finetuning FLAVA
+============================================
+"""
+
+######################################################################
+# Multimodal AI has recently become very popular owing to its ubiquitous
+# nature, from use cases like image captioning and visual search to more
+# recent applications like image generation from text. **TorchMultimodal
+# is a library powered by Pytorch consisting of building blocks and end to
+# end examples, aiming to enable and accelerate research in
+# multimodality**.
+# 
+# In this tutorial, we will demonstrate how to use a **pretrained SoTA
+# model called** `FLAVA <https://arxiv.org/pdf/2112.04482.pdf>`__ **from
```

Comment:
An overview of the FLAVA model would be helpful here (maybe with Fig. 2 in the paper?)